pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -205,6 +222,28 @@ func (s *Server) watchRootCertAndGenKeyCert(stop <-chan struct{}) {
 	}
 }
 
+func (s *Server) watchDNSCertForK8sCA(stop <-chan struct{},
```

Comment:
nit: rename to be more obvious. maybe call something like `RotateDNSCertForK8sCA`

Reply:
Done, thank you.